.gitignore

```
@@ -1 +1,3 @@
 node_modules/
+# We don't want to commit package locks for example projects
+**/package-lock.json
```

Nit: I think the comment is implied by what the code does. And there's really only one layer deep that they could be added, right?

```suggestion
packages/*/package-lock.json
```

IMO it's good to have more precise ignores to make it clear exactly what they're ignoring.